.github/workflows/connector-ci-tests.yml

```diff
@@ -80,9 +80,11 @@ jobs:
         if: matrix.connector
         uses: actions/checkout@v4
         with:
+          # In order to work correctly on forks, `repository` must be set if `ref` is set:
+          repository: ${{ github.repository }}
           # Java `integrationTestJava` Gradle task still calls Airbyte-CI, which fails on a detached head.
           # TODO: Remove the Airbyte-CI dependency for running java integration tests.
-          ref: ${{ github.head_ref }}
+          ref: ${{ github.head_ref || github.ref_name }}
```

Comment:
This change isn't strictly needed, but it makes the checkout action more portable. Other triggers besides `pull_request` may provide `ref_name` but not `head_ref`.